### DISCUSSION TITLE
Is there an intended pronunciation?

### DISCUSSION THREAD

**Discussion by djeikyb** [tauri-apps/tauri#6929] [Q&A]
How do ya'll pronounce "tauri"? How are _we all_ supposed to pronounce it lol? Is it like "ford taurus" but with an "ee" noise instead of an "us" noise?

**Comment by FabianLars** [comment]
Here's an example from 2 years ago: https://youtu.be/UxTJeEbZX-0?t=585 (if you end up watching the whole video, a lot of things mentioned there have changed over time and it may not represent the current state anymore)

And just for fun here's how i pronounce it with my intense german accent: https://cdn.discordapp.com/attachments/731495028677148753/969550370151997460/ttsMP3.com_VoiceText_2022-4-29_12_46_51.mp3 (using a german TTS voice)

**Comment by simonhyll** [comment]
Tauri comes from smashing the project "Tao" and "Wry" together. Tao-wry. Tau-ri. Tauri.
I think the phonetic notation for it would be `/taʊri/` but I'm no expert.
